BuildOptimizedImages/src/Web/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM microsoft/aspnetcore-build:1.0.1
+
+RUN mkdir /app
+WORKDIR /app
+
+ADD project.json .
+RUN dotnet restore --no-cache
```

Comment:
Ah. That makes sense. Yeah, we probably should talk about clarifying the descriptions and such if it is still the same.